public/app/features/folders/components/NewDashboardsFolder.tsx

```diff
@@ -37,8 +37,8 @@ export class NewDashboardsFolder extends PureComponent<Props> {
       .then(() => {
         return true;
       })
-      .catch(() => {
-        return 'Folder already exists.';
+      .catch(e => {
+        return e.message;
```

Comment:
Really like this improvement! :)
![Screenshot from 2020-04-08 10-23-23](https://user-images.githubusercontent.com/1438972/78761836-6deac300-7983-11ea-99b0-4025ebc3a890.png)

Reply:
this is not possible. See the referenced issue